### PR TITLE
Added support for writing to CMEK managed tables

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Review by suryasoma — APPROVED** [review]

File: README-template.md [hunk 1]
```diff
@@ -790,6 +790,19 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read/Write</td>
    </tr>
+  <tr>
+     <td><code>kmsKeyName</code>
```
Comment (emkornfield) [inline README-template.md:794]: maybe a more descriptive name specifying its purpose (I don't know if there will be other kms keys required for the connector in the future?

**Review by emkornfield — APPROVED** [review]

File: README-template.md (hunk 1 above)
Comment (davidrabinowitz) [reply README-template.md]: Renamed kmsKeyName to destinationTableKmsKeyName

**Pull request merged**